cookiecutter.json

```diff
@@ -15,5 +15,6 @@
   "maximum_supported_python_version": "3.13",
   "project_needs_c_code_compiled": ["No", "Yes"],
   "project_has_gui_tests": ["No", "Yes"],
+  "_is_update": "No",
```

Comment:
just a quick question, not sure whether `is_update` clear... since it's a public facing file. 

`_is_package_update_command` something like that?

Reply:
or since we are using a statement like `project_has_gui_tests`,

maybe `project_is_updated` ?